engine/__init__.py

```diff
@@ -17,45 +17,58 @@ class DreamTexturesNodeCategory(nodeitems_utils.NodeCategory):
     def poll(cls, context):
         return context.space_data.tree_type == DreamTexturesNodeTree.__name__
 
+pipeline_items = [
+    nodeitems_utils.NodeItem(NodeStableDiffusion.bl_idname),
+    nodeitems_utils.NodeItem(NodeControlNet.bl_idname),
+]
+
+input_items = [
+    nodeitems_utils.NodeItem(NodeInteger.bl_idname),
+    nodeitems_utils.NodeItem(NodeString.bl_idname),
+    nodeitems_utils.NodeItem(NodeImage.bl_idname),
+    nodeitems_utils.NodeItem(NodeCollection.bl_idname),
+    nodeitems_utils.NodeItem(NodeRenderProperties.bl_idname),
+]
+if bpy.app.version >= (3, 5, 0):
+    input_items.append(nodeitems_utils.NodeItem(NodeImageFile.bl_idname))
+
+utility_items =[
+    nodeitems_utils.NodeItem(NodeMath.bl_idname),
+    nodeitems_utils.NodeItem(NodeRandomValue.bl_idname),
+    nodeitems_utils.NodeItem(NodeRandomSeed.bl_idname),
+    nodeitems_utils.NodeItem(NodeSeed.bl_idname),
+    nodeitems_utils.NodeItem(NodeClamp.bl_idname),
+    nodeitems_utils.NodeItem(NodeFramePath.bl_idname),
+    nodeitems_utils.NodeItem(NodeCropImage.bl_idname),
+    nodeitems_utils.NodeItem(NodeJoinImages.bl_idname),
+    nodeitems_utils.NodeItem(NodeColorCorrect.bl_idname),
+    nodeitems_utils.NodeItem(NodeSeparateColor.bl_idname),
+    nodeitems_utils.NodeItem(NodeCombineColor.bl_idname),
+    nodeitems_utils.NodeItem(NodeSwitch.bl_idname),
+    nodeitems_utils.NodeItem(NodeCompare.bl_idname),
+    nodeitems_utils.NodeItem(NodeReplaceString.bl_idname),
+]
+if bpy.app.version >= (3, 5, 0):
+    utility_items.append(nodeitems_utils.NodeItem(NodeResizeImage.bl_idname))
+
+annotations_items = [
+    nodeitems_utils.NodeItem(NodeAnnotationDepth.bl_idname),
+    nodeitems_utils.NodeItem(NodeAnnotationNormal.bl_idname),
```

Comment:
@carson-katri This is from your changes that are on `main` https://github.com/carson-katri/dream-textures/commit/66a148518eb8a697e1b8b4e3e08937b20cf78ce5. We can remove this from this PR if you'd like, but it will need to be added back in anyways before this lands on `main`

Reply:
I'd prefer this to be removed, I'll rebase the other branch.